### PR TITLE
Feat/handle projection integration to cardano services

### PR DESCRIPTION
**Pull request by lucas-barros**
# Context

_Reason for the change? If an issue exists, reference it here using a [keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)_

# Proposed Solution

# Important Changes Introduced

### CODE REVIEW

**Review by rhyslbw — DISMISSED** [review]
LGTM!

File: packages/cardano-services/src/Projection/prepareTypeormProjection.ts [hunk 1]
```diff
@@ -91,6 +116,7 @@ const entityInterDependencies: Partial<Record<EntityName, EntityName[]>> = {
   asset: ['block'],
   blockData: ['block'],
   currentPoolMetrics: ['stakePool'],
+  handle: ['block', 'tokens', 'asset', 'output'],
```
Comment (iccicci) [inline packages/cardano-services/src/Projection/prepareTypeormProjection.ts:119]: Looking at the migrations I would say that `handle` depends only on `tokens` while `tokens` depends on `asset` and `output`.
Should we make this reflect the dependencies in the migrations or is there something I'm missing?

Comment (mkazlauskas) [reply packages/cardano-services/src/Projection/prepareTypeormProjection.ts]: Nice catch, the entity actually only depends on AssetEntity, but `storeHandles` depends on all those other ones. Made a fixup to make it more accurate 1507e0361a2b1f7e0a217d45def76c01f8950b6f (the end result is the same though)

**Review by rhyslbw — APPROVED** [review]
Ready for squashing

**Review by mkazlauskas — APPROVED** [review]
:rocket:

**Review by iccicci — APPROVED** [review]
LGTM

**Pull request merged**